src/main/controllers/features/unincorporated/unincorporatedWhatIsYourRoleController.ts

```diff
@@ -1,25 +1,26 @@
-import { NextFunction, Request, Response, Router } from "express";
-import * as config from "../../../config";
+import { Session } from "@companieshouse/node-session-handler";
+import { NextFunction, Request, Response } from "express";
 import { validationResult } from "express-validator";
+import { ACSP_TYPE, BUSINESS_NAME } from "../../../common/__utils/constants";
+import * as config from "../../../config";
+import { ACSPData } from "../../../model/ACSPData";
+import { BASE_URL, STOP_NOT_RELEVANT_OFFICER, UNINCORPORATED_WHAT_IS_THE_BUSINESS_NAME, UNINCORPORATED_WHAT_IS_YOUR_ROLE, UNINCORPORATED_WHICH_SECTOR } from "../../../types/pageURL";
+import { addLangToUrl, getLocaleInfo, getLocalesService, selectLang } from "../../../utils/localise";
 import { FormattedValidationErrors, formatValidationError } from "../../../validation/validation";
-import { BASE_URL, UNINCORPORATED_WHAT_IS_THE_BUSINESS_NAME, UNINCORPORATED_WHAT_IS_YOUR_ROLE, UNINCORPORATED_WHICH_SECTOR, STOP_NOT_RELEVANT_OFFICER } from "../../../types/pageURL";
-import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
-import { Session } from "@companieshouse/node-session-handler";
-import { ACSP_TYPE, UNINCORPORATED_BUSINESS_NAME } from "../../../common/__utils/constants";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
     const lang = selectLang(req.query.lang);
     const locales = getLocalesService();
     const session: Session = req.session as any as Session;
     const acspType = session?.getExtraData(ACSP_TYPE)!;
-    const unincorporatedBusinessName = session?.getExtraData(UNINCORPORATED_BUSINESS_NAME)!;
+    const businessName : ACSPData = session?.getExtraData(BUSINESS_NAME)!;
```

Comment:
Is this correct? Business name of type ACSPData? I would have thought this would be string or something?